user-service/src/main/java/rs/edu/raf/banka/user_service/controller/UserController.java

```diff
@@ -27,6 +29,32 @@ public class UserController {
     @Autowired
     private final UserService userService;
 
+    @PostMapping("/agent/create")
+    @ApiOperation("Creates new agent")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "OK", response = Agent.class)})
+    public ResponseEntity<?> createNewAgent(@RequestBody CreateAgentForm createAgentForm){
+        return ResponseEntity.ok().body(userService.createAgent(createAgentForm));
+    }
+
+
+    @PatchMapping("/limit-change/{id}")
+    @ApiOperation("Changes agent's limit")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "OK", response = Agent.class)})
+    public ResponseEntity<?> changeLimit(@PathVariable long id, @RequestBody ChangeLimitForm changeLimitForm){
+        Agent agent = userService.getAgentById(id);
+        userService.changeLimit(agent, changeLimitForm.getLimit());
+        return ResponseEntity.ok().build();
+    }
+
+    @PatchMapping("/limit-reset/{id}")
+    @ApiOperation("Resets agent's limitUsed")
+    @ApiResponses(value = {@ApiResponse(code = 200, message = "OK", response = Agent.class)})
+    public ResponseEntity<?> resetLimit(@PathVariable long id){
+        Agent agent = userService.getAgentById(id);
+        userService.resetLimitUsed(agent);
+        return ResponseEntity.ok().build();
+    }
```

Comment:
Agent je tip korisnika, nema nikakve potrebe da se modeluje na zaseban način, da se dodaju nove API rute i ostalo. Ovo će mnogo da oteža implementaciju frontend timu.